projects/m1/005-units-of-time-again/js/index.js

```diff
@@ -0,0 +1,44 @@
+// inserimento input utente
+let totalInputSeconds = 673876543;
```

Comment:
This value ​​should not be set by you but **requested by the user**.
you can use `prompt-sync` like this:
`const prompt = require('prompt-sync')();`
or
`const prompt = require('prompt-sync')({sigint: true});`